gallery/src/styles/NxPageTitle/NxPageTitlePage.tsx

```diff
@@ -49,10 +53,31 @@ const NxPageTitlePage = () =>
             </td>
           </tr>
           <tr className="nx-table-row">
-            <td className="nx-cell"><code className="nx-code">.nx-page-title__page-icon</code></td>
+            <td className="nx-cell"><code className="nx-code">.nx-page-title__headings</code></td>
+            <td className="nx-cell"></td>
```

Comment:
So no convenience component for this then?

Reply:
> So no convenience component for this then?

https://github.com/sonatype/sonatype-react-shared-components/pull/363/commits/320049f434ad835a62447b37f287478041e74fe5